bleak/backends/dotnet/client.py

```diff
@@ -739,7 +739,7 @@ def dotnet_notification_parser(sender: Any, args: Any):
         reader.Dispose()
 
         return loop.call_soon_threadsafe(
-            func, sender.Uuid.ToString(), bytearray(output)
+            func, int(sender.AttributeHandle), bytearray(output)
```

Comment:
is AttributeHandle not already an int? winrt docs say it returns ushort.

Reply:
It is. I never trust `pythonnet` to give me what I expect though. In the cases where I have not debugged it thoroughly, I tend to add a extra conversion just in case I get a `UInt32` object or similar instead.

Debugged and checked this out now. It was an `int`.